wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -187,10 +187,12 @@
 
 <?php else : ?>
 	<section>
-		<div class="row align-middle between section-heading section-heading--with-space">
-			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+		<div class="row align-right">
 			<?php get_template_part( 'template-parts/component', 'archive-search' ); ?>
```

Comment:
Moving the search before the title changes the mobile view

<img src="https://github.com/WordPress/Learn/assets/1017872/c0ef4d72-11ae-4b20-9572-46d845ab2933" width="400">